operators/azure/cloud_ops_registration.cc

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "ocos.h"
+#include "azure_invokers.hpp"
+#include "openai_invokers.hpp"
+
+#ifdef AZURE_INVOKERS_ENABLE_TRITON
+#include "azure_triton_invoker.hpp"
+#endif
+
+using namespace ort_extensions;
+
+const std::vector<const OrtCustomOp*>& AzureInvokerLoader() {
+  static OrtOpLoader op_loader(CustomAzureStruct("AzureAudioToText", AzureAudioToTextInvoker),
+                               CustomAzureStruct("AzureTextToText", AzureTextToTextInvoker),
+                               CustomAzureStruct("OpenAIAudioToText", OpenAIAudioToTextInvoker)
+#ifdef AZURE_INVOKERS_ENABLE_TRITON
+                                   ,
+                               CustomAzureStruct("AzureTritonInvoker", AzureTritonInvoker)
+#endif
+#ifdef TEST_AZURE_INVOKERS_AS_CPU_OP
+                                   ,
+                               CustomCpuStruct("AzureAudioToText", AzureAudioToTextInvoker),
+                               CustomCpuStruct("AzureTextToText", AzureTextToTextInvoker),
```

Comment:
Can we just use CPUExecutionProvider for these custom ops? Not sure what AzureExecutionProvider is required for.

Reply:
Yes, we can and we did - the ops will be registered both as CPU and AzureEP ops.